src/prune_builds

```diff
@@ -39,6 +39,7 @@ def write_json(path, data):
 
 # Handle --insert-only
 if args.insert_only:
+    builddata = {}
```

Comment:
Won't this line fail as well?

```
builddata['builds'].insert(0, args.insert_only)
```

Let's try out this suggestion feature:

```suggestion
    builddata = {'builds': []}
```

Reply:
Suggestions seem cool, but it's too bad it doesn't support turning into `fixup!` or so.